core/modules/user/user.module

```diff
@@ -1268,6 +1268,16 @@ function user_menu() {
     'file' => 'user.admin.inc',
   );
 
+  // Login settings.
+  $items['admin/config/people/login'] = array(
+    'title' => 'Login settings',
+    'description' => 'Configure login settings',
```

Comment:
I think these `login` uses are fine. As the description can be displayed in admin pages, this should be a sentence like `Configure the way visitors may log in and limit log in attempts.`